src/worker.js

```diff
@@ -1,9 +1,10 @@
 /* eslint-disable import/first */
-require('newrelic');
+// require('newrelic');
```

Comment:
I think this should probably not be commented out.

Reply:
you are, of course, correct